src/lib/libhtml5_webgl.js

```diff
@@ -331,9 +331,12 @@ var LibraryHtml5WebGL = {
   },
 
   emscripten_webgl_destroy_context__proxy: 'sync_on_webgl_context_handle_thread',
-  emscripten_webgl_destroy_context__deps: ['free'],
+  emscripten_webgl_destroy_context__deps: ['free', '$JSEvents'],
   emscripten_webgl_destroy_context: (contextHandle) => {
     if (GL.currentContext == contextHandle) GL.currentContext = 0;
+    // Release all JS event handlers on the DOM element that the GL context is
```

Comment:
Should we keep the soft dependency in the form of `if (typeof JSEvents == 'object') {` rather than adding a hard dependency?

Or is the code in the tile all expected to to depend in libhtml5 stuff already?

Reply:
sgtm

Reply:
Hmm.. but actually it looks like `emscripten_webgl_do_create_context` should not be depending on `JSEvents` .. I don't see any usage of it in that function.

Reply:
Ah, you're right. That's weird. It looks like it's only used by `$registerWebGlEventCallback()` via the 
`emscripten_set_webglcontextlost_callback()` and `emscripten_set_webglcontextrestored_callback()` APIs.